### ISSUE TITLE
Some inputs have wrong height on mobile browsers

### ISSUE THREAD

**Issue by gabbsmo** [n8design/htwoo#150]
**Describe the bug**
`<select>` and `<input type="date">` looks OK in desktop browser, but does not have the same height as `<input type="text">` on some mobile browsers.

**To Reproduce**
`<select class="hoo-input-text><!-- omitted --></select>`
`<input type="date" class="hoo-input-text">`

**Expected behavior**
Same height as `<input type="text" class="hoo-input-text">`

**Smartphone:**
 - Device: Samsung Galaxy S1
 - OS: Android
 - Browser: Samsung Internet

**Additional context**
I realize that there is a `hoo-input-date` class as well, but found that it missed some functionality from `hoo-input-text`, or I was using it wrong. Either way, `hoo-input-text` did everything I wanted out of the box, except for this issue.

I know the `<select>` tag is not in the style guide so technically not a bug, but works just as well as the date field I mentioned before. I can open a separate feature request issue for it if you want.

My workaround:
```
select.hoo-input-text,
input[type="date"].hoo-input-text {
    height: 1.875rem;
}
```

Fixed height is OK, since these field types are not multi-line.

**Comment by StfBauer** [comment]
Are you sure the base font size is set to 1rem at 16px?

**Comment by gabbsmo** [comment]
Font size for `.hoo-input-text` is 0.875rem. Font size on html/body is undefined in CSS in my case so it defers to my browser settings.

**Comment by StfBauer** [comment]
Yes but if you change the font of your OS that the browser shows smaller text, this changes the font size on root. There is now browser default other then the default setting is set to 16px. Can you share a screenshot?

**Comment by gabbsmo** [comment]
Sure thing.

![Screenshot_20241104_130228_Samsung Internet](https://github.com/user-attachments/assets/554a6359-0bf1-4495-bff4-cbee93ab8f38)
![Screenshot_20241104_130245_Samsung Internet](https://github.com/user-attachments/assets/4c01c70d-05d6-4f55-936e-95a76e143a71)

**Comment by gabbsmo** [comment]
Indeed. Luckily, setting the height of the control works in this case. Also, font is not matching, but at least it looks better when height is matching.

**Comment by StfBauer** [comment]
I need to find an Android emulator to check what is going on there.

**Comment by gabbsmo** [comment]
I frequently use BrowserStack for device testing. They have a sponsorship program for open source projects. https://www.browserstack.com/open-source

**Comment by StfBauer** [comment]
Thanks haven't tought about that ;) - Submitted it now.
